dbt_platform_helper/commands/config.py

```diff
@@ -156,21 +158,26 @@ def deployment():
     exit(0 if compatible else 1)
 
 
-def tool_versions():
+def _check_tool_versions_and_return_platform_helper_versions(platform_helper_versions):
```

Comment:
I think this still wants renaming.

Reply:
Or the function split in two? Have one bit check tool versions and print the table. The other could return the versions

Reply:
I would prefer the getting of the versions to be separate from the checking of the versions.

Reply:
The versions are now passed into this function and I renamed it back to "_check_tool_versions"

Reply:
@gabelton I may rework this method some more when I bring in the configurable platform-tools version.

Reply:
Is it worth moving the get AWS and Copilot versions calls outside of the function?

I think it would then only be responsible to the checking.

Reply:
Sure. 2 ticks